src/app/test-record/test-history/test-history.component.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
start to write a snapshot test and see if you templates are generated accordingly based on the data fed